src/RacingCarGame.js

```diff
@@ -0,0 +1,79 @@
+import { Console, Random } from "@woowacourse/mission-utils";
+import { ERROR_MESSAGES } from "./constants/errorMessages.js";
+
+class RacingCarGame {
+  static CAR_NAME_SEPARATOR = ",";
+
+  static MAX_CAR_NAME_LENGTH = 5;
+
+  static MIN_CAR_NAME_LENGTH = 1;
+
+  static MIN_SCORE_TO_MOVE = 4;
+
+  validateCarName(carName) {
+    return (
+      carName.length >= RacingCarGame.MIN_CAR_NAME_LENGTH &&
+      carName.length <= RacingCarGame.MAX_CAR_NAME_LENGTH
+    );
+  }
+
+  validateRoundCount(roundCount) {
+    return (
+      !isNaN(roundCount) &&
+      Number.isInteger(Number(roundCount)) &&
+      Number(roundCount) > 0
+    );
+  }
```

Comment:
isInteger 메서드가 isNaN의 역할을 한다고 생각합니다! 왜냐하면 정수가 아닌게 숫자일리는 없자나요??
또 Number(roundCount)가 반복되니 이것도 수정하면 좋을 거 같습니당
```suggestion
  validateRoundCount(roundCount) {
  const numberRoundCount = Number(roundCount)
    return (
      Number.isInteger(numberRoundCount ) &&
      numberRoundCount  > 0
    );
  }
```

Reply:
오 좋은 제안 감사합니다!
중복 역할을 하는 코드들을 간소화할 수 있겠네요~~!!